bitnami/cassandra/4.0/debian-11/rootfs/opt/bitnami/scripts/libcassandra.sh

```diff
@@ -855,7 +855,7 @@ wait_for_nodetool_up() {
         # https://community.datastax.com/questions/13764/java-version-for-cassandra-3113.html
         local -r check_cmd=("${CASSANDRA_BIN_DIR}/nodetool" "-Dcom.sun.jndi.rmiURLParsing=legacy")
         local -r check_args=("status" "--port" "$CASSANDRA_JMX_PORT_NUMBER")
-        local -r machine_ip="$(dns_lookup "$CASSANDRA_HOST" "v4")"
+        local -r machine_ip="$(dns_lookup "${CASSANDRA_BROADCAST_ADDRESS:-$CASSANDRA_HOST}" "v4")"
```

Comment:
Hey! Thanks for your contribution!

I think that this would also apply to [L842](https://github.com/bitnami/containers/blob/16398787d2b648d2799a8ad4122b3686e7c8e97d/bitnami/cassandra/4.0/debian-11/rootfs/opt/bitnami/scripts/libcassandra.sh#L842)

Reply:
Makes absolutely sense... ;)
Fixed in  0a6baed